docs/introduction.rst

```diff
@@ -0,0 +1,15 @@
+
+Introduction
+============
+
+pysatMissions allows users to run build simulated satellites for TLE info and add empirical data. It includes the pysat_sgp4 and pysat_ephem instrument modules which can be imported into pysat.
+
+Main Features
+-------------
+- Simulate satellite orbits from TLEs and add data from empirical models
+- Import ionosphere and thermosphere values through pyglow
+- Import coordinates through apexpy
+- Import magnetic coordinates through aacgmv2
+- Import geomagnetic basis vectors through pysatMagVect
+
+This document covers installation, a tutorial on pysatMissions including demonstration code, and an API reference.
```

Comment:
It would also be good to have a page/section that includes references to the apex/AACGMV2 papers, MSIS, IRI, etc.

Reply:
added a references document